docs/docsite/rst/playbook_guide/playbooks_reuse_roles.rst

```diff
@@ -281,63 +281,98 @@ role ``meta/argument_specs.yml`` file. All fields are lower-case.
     * This should be ``main`` in the case of an unspecified entry point.
     * This will be the base name of the tasks file to execute, with no ``.yml`` or ``.yaml`` file extension.
 
-    :short_description:
+.. list-table:: Role Argument Specification Fields
+  :width: 90%
+  :widths: auto
+  :header-rows: 1
+
+    * - Name
+      - Description
+      - Type
+      - Required
+      - Default
+    * - short_description
+      - A short, one-line description of the entry point.
+      - str
+      - Yes
+      -  
+    * - description
+      - A longer description that may contain multiple lines.
+      - str
+      - No (see notes)
+      - null
+    * - author
+      - Name of the entry point authors.
+      - list\n elements: str
+      - No
+      - []
+    * - options
+      - Options are often called "parameters" or "arguments".
+      - dict (see table below)
+      - No
+      - {}
+
+
+Options themselves have a defined structure:
+.. list-table:: Role Argument Specification options
+  :width: 90%
+  :widths: auto
+  :header-rows: 1
+
+    * - Name
+      - Description
+      - Type
+      - Required
+      - Default
+    * - name
```

Comment:
This is a bit confusing, as there is no field called `name` to my knowledge. The option's name is the key of the option dict.